src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -1,15 +1,68 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.Azure.Management.Monitor</PackageId>
     <Description>Microsoft Azure Monitor Library</Description>
     <VersionPrefix>0.16.0-preview</VersionPrefix>
-    <AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
+    <AssemblyName>Microsoft.Azure.Monitor</AssemblyName>
```

Comment:
Our previous assembly name (and Swagger spec) was confused with AppInsights and perhaps other assemblies. So after some requests from other teams we changed the name.
Also, our team is called Monitor after the last re-org, before we were the Insights team.